packages/dht/src/connection/WebRTC/ParsedLocalCandidate.ts

```diff
@@ -0,0 +1,39 @@
+export enum CandidateType {
+    HOST = 'host',
+    SRFLX = 'srflx',
+    PRFLX = 'prflx',
+    RELAY = 'relay'
+}
+
+export class ParsedLocalCandidate {
```

Comment:
I think a class here is a bit overblown since what we want to achieve is to replace the IP address when the candidateType is `host`?

Something like
```ts
replaceIpIfCandidateTypeIsHost(candidate: string, replacementIp: string): string {
    if (candidate.endsWith('host')) {
        const [_ip, ...rest] = candidate.split(' ')
        return [replacementIp, ...rest].join(' ')
    }
    return candidate
}
```